ports-wip/rsync/patches/log.c.patch

```diff
@@ -0,0 +1,17 @@
+--- log.c.orig	2018-02-28 01:37:31 +0100
```

Comment:
@berryamin please delete rsync from ports-wip, we have the same version in ports

Reply:
done